src/common/lpspi.rs

```diff
@@ -493,6 +502,25 @@ impl<P, const N: u8> Lpspi<P, N> {
         Interrupts::from_bits_truncate(ral::read_reg!(ral::lpspi, self.lpspi, IER))
     }
 
+    /// Set the sampling point of the LPSPI in master mode.
+    ///
+    /// When set to `SamplePoint::DelayedEdge`, the LPSPI master will sample the input data
+    /// on a delayed LPSPI_SCK edge, which improves the setup time when sampling data.
+    ///
+    /// On [`init`](`Lpspi::init`), the `SamplePoint::DelayedEdge` congfiguration is set.
```

Comment:
```suggestion
    /// This driver sets `SamplePoint::DelayedEdge` by default.
```

Reply:
We could also attach this note to the documentation of [`new`](https://github.com/imxrt-rs/imxrt-hal/blob/8e254826866d77e080979a53bf6d956c9941cbb4/src/common/lpspi.rs#L369-L371), describing it with the SPI mode and clock settings.